src/PrestaShopBundle/Controller/Admin/Improve/International/TranslationsController.php

```diff
@@ -52,11 +53,14 @@ class TranslationsController extends FrameworkBundleAdminController
     public const controller_name = self::CONTROLLER_NAME;
 
     /**
-     * @Template("@PrestaShop/Admin/Translations/overview.html.twig")
+     * Renders the translation page
      */
     public function overviewAction()
     {
-        return parent::overviewAction();
+        return $this->render('@PrestaShop/Admin/Improve/International/Translations/overview.html.twig', [
+            'is_shop_context' => (new Context())->isShopContext(),
+            'layoutTitle' => empty($this->layoutTitle) ? '' : $this->trans($this->layoutTitle, 'Admin.Navigation.Menu'),
```

Comment:
FYI I copied this over from `FrameworkBundleAdminController::overviewAction` so that I can deprecate that function.